DN2/docs/demo.jl

```diff
@@ -0,0 +1,68 @@
+#' # DN2 - Porazdelitvena funkcija normalne slučajne spremenljivke, Ploščina Bézierove krivulje; Matija Ojo; 63200205
+
+#' ## Porazdelitvena funkcija normalne slučajne spremenljivke 
+
+#' Porazdelitvena funkcija normalne slučajne spremenljivke je definirana kot:
+
+#' $$F(x) = \frac{1}{\sqrt{2\pi}} \int_{-\infty}^{x} e^{-\frac{t^2}{2}} dt$$.
+
+#' V praksi se uporablja za izračun verjetnosti, da bo vrednost slučajne spremenljivke padla v določen interval.
+
+#' Za izračun porazdelitvene funkcije normalne slučajne spremenljivke se uporablja le numerične metode, saj integrala ne moremo izračunati analitično.
+#' V nalogi je uporabljena [adaptivna Simpsonova metoda](https://en.wikipedia.org/wiki/Adaptive_Simpson%27s_method) za izračun integrala.
+#' Adaptivna Simpsonova metoda je osnovana na [Simpsonovem pravilu](https://en.wikipedia.org/wiki/Simpson%27s_rule), ki numerično izračuna vrednost integrala tako da evalvira vrednost funckije na vmesnih točkah.
+#' Velja: $\int_a^b = \frac{b - a}{6} (f(a) + 4f(\frac{a+b}{2}) + f(b))$.
+#' Adaptivna metoda pa razdeli interval na dva dela in izračuna integral na vsakem delu posebej, in njuno vsoto uporabi kot naslendji približek.
+#' Če je razlika med dobljenim pribiližkom integrala in prejšnjim približkom večja od določene tolerance, se interval razdeli na še manjše dele, sicer vrne izračunani približek.
+
+#' ### Rezultati
+
+#' Poglejmo si graf porazdelitvene funkcije normalne slučajne spremenljivke za različne vrednosti $x$,
+#' pri čemer je vsaka vrednost funckije izračunana z adaptivno Simpsonovo metodo.
+
+using DN2
+using Plots
+
+xs = range(-4, stop=4, length=1000)
```

Comment:
Namesto grafa funkcije bi bilo bolje narisati graf napake in opisati, kako ste ocenili napako.